packages/picker/src/spectrum-picker.css

```diff
@@ -687,7 +687,7 @@ governing permissions and limitations under the License.
         )
     );
 }
-#button.is-loading .picker {
+:host([pending]) #button .picker {
```

Comment:
`pending` in Button delays, which makes sense for Button, we're attempting to cede user interface to processing. Here, the `pending` would likely come from an unrelated interaction, so does it make any sense to delay here as well? @TarunAdobe

Reply:
I agree Picker should doesn't need to delay the `pending` effect.